Fix sorts_query_result -> sort_query_result, add test coverage for strain circuits, input levels, and dictionary lookups.

mwes opened the pull request:
Resolves: https://github.com/SD2E/synbiohub_adapter/issues/70

```
python -m unittest tests/test_sparql_queries.TestSBHQueries                              
Password: 
................................................
----------------------------------------------------------------------
Ran 48 tests in 33.042s

OK
```

mwes commented:
Oh bugger. It would appear I've introduced 7 `pycodestyle` errors. It's unclear from the build failure what these are. I'll need to track them down and diff against the existing errors to get this to pass.

bbartley commented:
@mwes you can diagnose style violations using the following:
```
VERBOSE=1 python3 -m unittest tests/test_pycodestyle.py
```
Note that you don't need to resolve the exact same 7 style violations you introduced, you just need to resolve _any_ 7 style violations.

mwes commented:
Thanks @bbartley I resolved the 7 style errors I introduced. Also fixed some build issues with the docker build + tests - it was using an older version of `pycodestyle` baked into the Ubuntu Python3 image.
